api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/models/FunctionalModelTest.kt

```diff
@@ -93,7 +93,7 @@ internal class FunctionalModelTest {
             correctTestModel.summary()
         )
 
-        val graphLines = correctTestModel.kGraph().toString().split('\n').toList()
+        val graphLines = correctTestModel.kGraph().toString().split('\n').toSet()
```

Comment:
Does this change fix the test? I thought the problem is in the graph operators

Reply:
Yes, the test is green. I think we should not rely on ordering here, since internally `toString()` uses `org.tensorflow.Graph::operations` which states:
> The order of iteration is unspecified

Reply:
Hmm, interesting that it was broken on this test, which could influence on a graph